source/src_pw/global.h

```diff
@@ -26,13 +26,253 @@
 #include "../src_parallel/ft.h"
 
 #ifdef __CUDA
+static const char *_cublasGetErrorEnum(cublasStatus_t error)
+{
```

Comment:
Are these functions globalfunctions? You would better add a namespace for them.

Reply:
I just added namespace CudaCheck and HipCheck to fix it.